raycicmd/converter.go

```diff
@@ -222,10 +222,6 @@ func (c *converter) convertGroups(gs []*pipelineGroup, filter *stepFilter) (
 	// Finalize the conversion.
 	var bkGroups []*bkPipelineGroup
 	for _, groupNode := range groupNodes {
-		if !groupNode.hit() {
-			continue
-		}
-
```

Comment:
a bug fix here.

Reply:
when group is rejected, all steps will be marked as rejected, and there is no need for this skipping here.

on the other hand, even when the group node is not hit, it might have steps in it that is hit as a dependency, so early skipping here is incorrect.